src/main/java/com/example/demo/model/Book.java

```diff
@@ -0,0 +1,58 @@
+package com.example.demo.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+import java.math.BigDecimal;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "books", uniqueConstraints = {@UniqueConstraint(columnNames = "isbn")})
```

Comment:
```suggestion
@Table(name = "books")
```
using column constraint is enough